modules/tinymce/src/themes/silver/test/ts/module/StickyHeaderUtils.ts

```diff
@@ -184,6 +185,18 @@ const assertEditorClasses = (docked: boolean) => {
   })), container);
 };
 
+const assertHeaderPosition = async (top: boolean, location: ToolbarLocation, value: number) => {
```

Comment:
Can we please change `top` -> `isToolbarTop` to keep it consistent with how it's used elsewhere here?

Alternatively, we could just pass `location` and then determine if we're positioning at the top using that.

Reply:
updated